### DISCUSSION TITLE
Nature progettuali OpenCup

### DISCUSSION THREAD

**Discussion by tbuon** [ondata/italian-public-sector-pnrr-data-guide#16] [Proposte]
### Testo

"Al momento su OpenCUP non sono pubblicate tutte le nature progettuali, ma soltanto quelle relative ai progetti di investimento pubblico “Lavori Pubblici, Incentivi alle imprese e Contributi per calamità naturali”.... Questo messaggio è errato, su OpenCup sono presenti anche le nature Cup legate ad acquisto di beni e servizi. Basta scaricarsi il file completo dei progetti e vedere nella colonna delle nature Cup. Solo nell'infografica di OpenCup vengono fatti dei focus sui lavori, incentivi e contributi.

**Comment by aborruso** [comment]
Ciao @tbuon,
innanzitutto, grazie per essere qui.

Quella nota deriva da quanto riportato sul [sito di OpenCUP](https://www.opencup.gov.it/portale/web/opencup/opendata). 

![image](https://user-images.githubusercontent.com/30607/236925190-f131b40d-ed76-4b07-9b13-6ce505ade90b.png)
